lib/locale/pl_PL/strings.php

```diff
@@ -4151,6 +4151,7 @@
 
 
 $_LANG['Removed:'] = 'Usunięte:';
+$_LANG['Submit and add new'] = 'Zapisz i dodaj nowy';
 $_LANG['Remover:'] = 'Usuwający:';
```

Comment:
Dziwnie to brzmi. Podejrzewam, że @awbnet najlepiej by to wymyślił :)

Reply:
**Zapisz i dodaj nowy** brzmi dziwnie? Hmm. W CRM-ach najczęściej spotykam się z **Zapisz i dodaj kolejny**.

Reply:
Tłumaczenie zmienione.

Reply:
Tylko nie znam kontekstu, bo jeśli to zgłoszenie to **Zapisz i dodaj kolejne**.

Reply:
Zapisz i dodaj kolejne brzmi lepiej. Zmieniłem.

Reply:
Super